src/constants/messaging.js

```diff
@@ -3,4 +3,4 @@ export const AUTH_ERROR = "Error authorizing you"; // Must match Go's miniware.A
 export const ON_CLOSE_RECONNECT_MESSAGE = "Message WebSocket closed. Reconnecting...";
 export const ONE_MINUTE = 60 * 1000;
 
-export const USER_STATUS_DELAY_MS = 10 * ONE_MINUTE;
+export const USER_STATUS_DELAY_MS = 2 * ONE_MINUTE;
```

Comment:
@Spetastian We're going to have thousands of people using this. And right now, the LeapChat Go code sends the most recent 50 "messages" to the client, where a message could be a user status _or_ a useful message. So even though a user's status will be out of date for an extra 8 minutes by having `USER_STATUS_DELAY_MS` set to 10 minutes rather than 2 minutes, that'll reduce load on our servers considerably, and also show more useful messages right when users connect.

Soon enough we'll have real accounts people can optionally sign into, in which case the server will know which WebSocket connection corresponds to which user, and thus have the server send a message to the relevant chat room users _right_ when a user disconnects.  That requires having the clients hand over a bit more responsibility/trust to the server, but not much, and it'll make this whole user status thing much more efficient.

Reply:
Of course, ill revert it.